ui/src/app/views/settings/component.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import { mount } from 'enzyme';
+import * as React from 'react';
+
+import {SettingsReact, SettingsState} from './component';
+
+import {Cluster, ClusterStatus} from 'generated/fetch/api';
+
+describe('SettingsComponent', () => {
+  let props: {};
+
+  const component = () => {
+    return mount<SettingsReact, {}, SettingsState>
+    (<SettingsReact {...props}/>);
```

Comment:
rm newline? Breaking between the method name and the paren of the method invocation is not very readable